package.json

```diff
@@ -40,6 +43,12 @@
   },
   "dependencies": {
     "@types/node-fetch": "^2.3.4",
+    "foundry-rpc": "^2.0.0",
```

Comment:
Why did you add this line?

Reply:
I did not add this, I think `npm init` do that.

Reply:
Let's remove it then.
All changed content must be made with your intent.